gc/base/standard/Scavenger.cpp

```diff
@@ -1641,12 +1642,28 @@ MM_Scavenger::updateCopyScanCounts(MM_EnvironmentBase* env, uint64_t slotsScanne
 {
 	env->_scavengerStats._slotsScanned += slotsScanned;
 	env->_scavengerStats._slotsCopied += slotsCopied;
-	uint64_t updateResult = _extensions->copyScanRatio.update(env, &(env->_scavengerStats._slotsScanned), &(env->_scavengerStats._slotsCopied), _waitingCount);
+	uint64_t updateResult = _extensions->copyScanRatio.update(env, &(env->_scavengerStats._slotsScanned), &(env->_scavengerStats._slotsCopied), _waitingCount, &(env->_scavengerStats._copyScanUpdates));
 	if (0 != updateResult) {
 		_extensions->copyScanRatio.majorUpdate(env, updateResult, _cachedEntryCount, _scavengeCacheScanList.getApproximateEntryCount());
 	}
 }
 
+MMINLINE void
+MM_Scavenger::flushCopyScanCounts(MM_EnvironmentBase* env, bool majorFlush)
+{
+	uint64_t updateResult = 0;
+
+	if(env->_scavengerStats._slotsScanned != 0) {
```

Comment:
keep spacing between the keyword and brackets. here, and

Reply:
Will do, just going to wait for the platform builds, which just got kicked off, to run and complete before pushing another commit to avoid a manual restart... not sure if pushing a commit will interfere and require a manual restart

Reply:
Yeah sorry, I jumped the gun on those builds. After the builds pass, if fixing formatting is the only change you push, I won't bother re-launching builds.